mixer/cmd/bundles.go

```diff
@@ -51,7 +51,13 @@ resultant list is written back out in sorted order.`,
 			if len(args) == 0 {
 				return errors.New("bundle add requires at least 1 argument if neither --all-local nor --all-upstream are passed")
 			}
-			bundles = strings.Split(args[0], ",")
+			// If just one arg passed, try splitting for compatibility
+			if len(args) == 1 {
```

Comment:
So I bet this is going to break with a weird error if someone does this:
```
mixer bundle add mixer,c-basic editors
```
But that is clearly user error at that point so 🚀

Reply:
Yup. It will will try to treat `mixer,c-basic` as a single bundle name, and error out saying it doesn't exist. I decided I was fine with this.

Reply:
Or, we can trivially solve this and not break on users missing a comma or adding a space with: 
```
        var list []string
        for _, arg := range args {
                list = append(list, strings.Split(arg, ",")...)
        }
        for i, item := range list {
                if item == "" {
                        list = append(list[:i], list[i+1:]...)
                }
        }
        bundles = list
```

Reply:
My gut says that this is a solution to a problem that doesn't really exist.

Reply:
FYI - I'm assuming this was just meant to be representative (so apologies if you already know this). but just in case, I wanted to mention that you can't strip values from a slice the way you are here. The items returned by `range` will not be updated when you modify the slice inside the loop, and thus this code will go out of bounds.

I just included a strip-empty-strings update to this PR that does the following:
```go
bundles = strings.Split(args[0], ",")
for i := 0; i < len(bundles); i++ {
	if bundles[i] == "" {
		bundles = append(bundles[:i], bundles[i+1:]...)
		i-- // Because removal shifts things left
	}
}
```

Reply:
@kevincwells 
Worth looking at https://github.com/golang/go/wiki/SliceTricks

In particular a better way to remove an item from a list if you don't care about the order is to copy the last item to the current position, set the last item to nil, so the garbage collector doesn't see a reference, and then shorten the slice by 1.

This is linear in the number of items to remove, rather than O(n^2)

Reply:
@icarus-sparry Yeah, that's pretty slick. Worth redoing given n here is quite small?

Definitely will keep in mind for the future either way.

Reply:
This is nice to know ^! Except for n usually being on the order of single or couple digits long I don't think we're losing that much performance here.